backend/server.js

```diff
@@ -1,20 +1,12 @@
 require("dotenv").config();
-const mongoose = require("mongoose");
-const Fastify = require("fastify");
+
 const createServer = require("./lib");
 const { config, validateConfig } = require("./config");
 
 validateConfig(config);
-const fastify = Fastify({ logger: true });
-const server = createServer(fastify);
-
-mongoose.connect(config.mongo.uri, config.mongo.params);
-mongoose.connection.on("error", (err) => {
-  server.log.error(err);
-  process.exit();
-});
+const server = createServer(config);
 
-server.listen(config.server.port, (err, address) => {
+server.listen(config.server.port, "0.0.0.0", (err, address) => {
```

Comment:
Why this "0.0.0.0", this totally unnecessary and you are adding more code for no reason. Don't agree with this change